android/src/main/java/com/codetrixstudio/capacitor/GoogleAuth/GoogleAuth.java

```diff
@@ -146,9 +143,49 @@ public void signOut(final PluginCall call) {
 
   @PluginMethod()
   public void initialize(final PluginCall call) {
+    String clientId = call.getString("androidClientId",
+            call.getString("clientId",
+                    this.getContext().getString(R.string.server_client_id)));
+
+    boolean forceCodeForRefreshToken = call.getBoolean("forceCodeForRefreshToken", false);
+
+    GoogleSignInOptions.Builder googleSignInBuilder = new GoogleSignInOptions.Builder(GoogleSignInOptions.DEFAULT_SIGN_IN)
+            .requestIdToken(clientId)
+            .requestEmail();
+
+    if (forceCodeForRefreshToken) {
+      googleSignInBuilder.requestServerAuthCode(clientId, true);
```

Comment:
Actually here you should use serverClientId

Reply:
As per docs: https://developers.google.com/identity/sign-in/android/offline-access